image_transport/CMakeLists.txt

```diff
@@ -16,7 +16,6 @@ find_package(Boost REQUIRED)
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES ${PROJECT_NAME}
-  DEPENDS message_filters pluginlib rosconsole roscpp roslib sensor_msgs
```

Comment:
This should be listed as `CATKIN_DEPENDS` instead of just removing this line, or building image_transport plugins fails (e.g. `compressed_depth_image_transport`) since cmake macro of depending packages is not exported any more with this change.